src/marqo/index.py

```diff
@@ -74,8 +75,9 @@ def create(config: Config, index_name: str,
             settings_dict: if specified, overwrites all other setting
                 parameters, and is passed directly as the index's
                 index_settings
-            inference_type: inference type for the index
-            storage_type: storage type for the index
+            inference_node_type: inference type for the index
+            storage_node_type: storage type for the index
+            inference_node_count: number of inference nodes for the index
```

Comment:
Can you please create unit tests for `inference_node_type`, `inference_node_count` and `storage_node_type` parameters, including edge cases (`None` or `0`)